match-lms-release/README.md

```diff
@@ -51,6 +53,50 @@ The release step will fail to write to `package.json` if you have branch protect
 [Learn how to set up the D2L_GITHUB_TOKEN...](https://github.com/BrightspaceUI/actions/blob/main/docs/branch-protection.md)
 
 ## Release Increments
+
 The first commit made during any given LMS release will update the Major and/or Minor versions to match the LMS version provided. Other commits during the same LMS release will trigger patch releases.
 
-Pull requests merged into maintenance branches created by the action (ex: `release/2022.2.x`) or named for a minor version (ex: `1.7.x`) will trigger patch releases for the maintenance branch version.
+Pull requests merged into maintenance branches created by the action (ex: `release/2022.2.x`) or named for a minor version (ex: `1.7.x`) or major version (ex: `1.x`) will trigger patch releases for the maintenance branch version.
+
+## NPM Package Deployment
+
+If you'd like the action to deploy your package to NPM, set the `NPM` option to `true`.
+
+NPM deployments for maintenance branches (ex: `release/2022.2.x`, `1.7.x`, or `1.x`) will be annotated with a tag corresponding to the branch version (ex: `release-2022.2.x`, `release-1.7.x`, or `release-1.x`). All other deployments will use NPM's default tag of `latest`.
+
+### CodeArtifact
+
+To publish to CodeArtifact, ensure that prior to running the `incremental-release` step that the [add-registry](https://github.com/Brightspace/codeartifact-actions/tree/main/npm) and the [get-authorization-token](https://github.com/Brightspace/codeartifact-actions/tree/main/get-authorization-token) steps have been run.
```

Comment:
Should this be `match-lms-release`?

Reply:
Might be good to include an example of this as well similar to what you did with NPM

```yml
- name: Get CodeArtifact authorization token
  uses: Brightspace/codeartifact-actions/get-authorization-token@main
  env:
    AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY_ID}}
    AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
    AWS_SESSION_TOKEN: ${{secrets.AWS_SESSION_TOKEN}}
- name: Add CodeArtifact npm registry
  uses: Brightspace/codeartifact-actions/npm/add-registry@main
  with:
    auth-token: ${{env.CODEARTIFACT_AUTH_TOKEN}}
```

Reply:
😄 you caught me copy-pasting the old docs @dlockhart had sent me...

Reply:
@devpow112 I've been doing some work in this area, and noticed the `env` in `auth-token: ${{env.CODEARTIFACT_AUTH_TOKEN}}`. Just wanted to confirm with you - is this correct, or should it be `secrets.CODEARTIFACT_AUTH_TOKEN` (or does it not matter)?

Thanks!